test/unittests.html

```diff
@@ -16,7 +16,7 @@
     <script src="lib/fetch.js"></script>
     <script>
         mocha.setup('bdd');
-        mocha.timeout(240000);
+        mocha.timeout(300000);
```

Comment:
ok if you prefer that, but i think that for what relates the performances regressions we should eventually take care of collecting the stats and already we are somehow doing that with travis/saucelabs.

what do you think?